pkg/webui/console/components/mac-settings-section/index.js

```diff
@@ -68,8 +68,14 @@ const m = defineMessages({
   classCTimeout: 'Class C timeout',
   maxDutyCycle: 'Maximum duty cycle',
   desiredMaxDutyCycle: 'Desired maximum duty cycle',
-  useAdr: 'Use adaptive data rate (ADR)',
   adrMargin: 'ADR margin',
+  adrUplinks: 'ADR number of transmissions',
+  adrAdaptiveDataRate: 'Adaptive data rate (ADR)',
+  adrDataRate: 'ADR data rate index',
+  adrTransPower: 'ADR transmission power index',
+  adrDynamic: 'Dynamic mode',
+  adrStatic: 'Static mode',
+  adrDisabled: 'Disabled',
```

Comment:
```suggestion
```
Don't add duplicate messages for things that can already be found in `sharedMessages`.